samwise.js

```diff
@@ -1,34 +1,34 @@
 /**
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.